doc/python_bindings.rst

```diff
@@ -16,6 +16,11 @@ installed as a single package called ``pydrake``.
    Please uninstall Anaconda or remove the Anaconda ``bin`` directory from the
    ``PATH`` before building or using the Drake Python bindings.
 
+.. warning::
+   On macOS, Drake only supports Python 3.8, which is located at
+   ``/usr/local/opt/python@3/bin/python3`` and is not usually on the
```

Comment:
This was not correct for me.  I needed `/usr/local/opt/python@3.8/bin`
@jwnimmer-tri  -- This was vicious enough that it probably needs to be very visible in the release notes.

Reply:
We have it as the very first announcement: https://github.com/jwnimmer-tri/drake/blob/release_notes-v0.17.0/doc/release_notes/v0.17.0.rst#announcements.  Can you comment on #13020 review where you'd like to see more?